test/test_jit.py

```diff
@@ -6781,6 +6781,149 @@ def forward(self, x):
 
         self.assertEqual(script_output, eager_output)
 
+    def test_breaks_continues(self):
```

Comment:
no tests check a combined break/continue loop. There also does not seem to be a test with a `if (x) .. break else: ...` pattern.